--- a/package/python-gobject/python-gobject.mk
+++ b/package/python-gobject/python-gobject.mk
@@ -14,11 +14,12 @@ PYTHON_GOBJECT_DEPENDENCIES = \
 	gobject-introspection \
 	host-pkgconf \
 	libglib2 \
-	python3
+	python3 \
+	python-pycairo
 
 PYTHON_GOBJECT_CONF_OPTS += \
-	-Dpycairo=disabled \
+	-Dpycairo=enabled \
 	-Dtests=false
 
 # A sysconfigdata_name must be manually specified or the resulting .so
--- a/package/python-pycairo/python-pycairo.mk
+++ b/package/python-pycairo/python-pycairo.mk
@@ -10,6 +10,7 @@ PYTHON_PYCAIRO_DEPENDENCIES = cairo
 PYTHON_PYCAIRO_LICENSE = LGPL-2.1 or MPL-1.1
 PYTHON_PYCAIRO_LICENSE_FILES = COPYING COPYING-LGPL-2.1 COPYING-MPL-1.1
+PYTHON_PYCAIRO_INSTALL_STAGING = YES
 
 PYTHON_PYCAIRO_CONF_OPTS = \
 	-Dpython=$(HOST_DIR)/bin/python \
